packages/slice-machine/package.json

```diff
@@ -32,7 +32,8 @@
     "slicemachine": "start-slicemachine --port 9999",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
     "lint:precommit": "eslint",
-    "audit": "better-npm-audit audit -l high -p -x 1054172",
+    "prettier:check": "prettier --check .",
```

Comment:
Does that mean the CI will only check formatting in `packages/slice-machine`, but not in all other packages? Yet, `lint-staged` formats every package so it doesn't seem to behave the same way :)

Reply:
You right I change that :)